Documentation/ColumnsConfig/CommonProperties/FieldInformation/AdminIsSystemMaintainer.rst

```diff
@@ -0,0 +1,50 @@
+..  include:: /Includes.rst.txt
+..  _tca_property_fieldInformation_adminIsSystemMaintainer:
+
+=======================
+adminIsSystemMaintainer
```

Comment:
Not really sure this needs documentation, it is only used by the Core and more an internal option. Can you think of any use case this can or should be used?

Reply:
You're right. Should not be part of TCA docs